pkg/network/cluster_config.go

```diff
@@ -159,6 +159,8 @@ func StatusFromOperatorConfig(operConf *operv1.NetworkSpec, oldStatus *configv1.
 		status.ClusterNetworkMTU = int(*operConf.DefaultNetwork.OpenShiftSDNConfig.MTU)
 	case operv1.NetworkTypeOVNKubernetes:
 		status.ClusterNetworkMTU = int(*operConf.DefaultNetwork.OVNKubernetesConfig.MTU)
+	case operv1.NetworkTypeKuryr:
+		status.ClusterNetworkMTU = int(*operConf.DefaultNetwork.KuryrConfig.MTU)
```

Comment:
This will explode on nil, wouldn't it? Checking it isn't necessary for other SDNs as both set a default [1][2]. We don't.

[1] https://github.com/openshift/cluster-network-operator/blob/b1abc131ba75de10deabb2b3e05bbfa74cbdb5ce/pkg/network/openshift_sdn.go#L200-L212
[2] https://github.com/openshift/cluster-network-operator/blob/6faec3b5d3615a37779f4825e9e0abd48db24204/pkg/network/ovn_kubernetes.go#L265-L273

Reply:
yes, it would. But, I made sure to fill the `KuryrConfig` with the MTU on the `renderKuryr`[1]. I needed to do there instead of fillDefaults cause we need the result of the Bootstrap.

[1] https://github.com/openshift/cluster-network-operator/pull/1043/commits/c6a86b08f3ef7a50e6a08a03cb5559e12ca7cf2b#diff-f66986e5dabe1ee056321bd3ff9bea98a698e49c8c25012f0af7327bfddac82eR123

Reply:
Alright, makes sense, thanks!